target_decisioning_engine/types/limited_key_dict.py

```diff
@@ -27,7 +27,7 @@ def _validate_kwargs_and_update(self, kwargs):
         """
         if not kwargs:
             return
-        valid = {key: value for (key, value) in kwargs.items() if key in self._get_valid_keys()}
+        valid = {key: value for (key, value) in list(kwargs.items()) if key in self._get_valid_keys()}
```

Comment:
I originally added that for backward compatibility with Python 2.7.  Python 3 returns an iterator from items(), and Python 2.7 returns a list.  But since this code is just doing an iteration anyway I don't think I need it; will remove.